### ISSUE TITLE
Gathering support

### ISSUE THREAD

**Issue by christianbundy** [fraction/oasis#268]
**What's the problem you want solved?** Events are hard to manage with regular posts, and the gathering subprotocol is very common and well-supported in other clients.

**Is there a solution you'd like to recommend?** Snag code from [this repo](https://github.com/ssbc/scuttle-gathering) and make a gathering page to collect them.

**Comment by black-puppydog** [comment]
We (that is @Powersource and me) wouldn't really want to recommend oasis over patchwork and then have to explain that gatherings are missing as a feature. So I changed the milestone to the "barebones" so we can potentially have someone work on it for monies. :)

**Comment by cbleslie** [comment]
Nicety - Gatherings are important, but I don't see them as a core feature for SSB? Though, that's just me. I don't know if other people find it critical.

**Comment by christianbundy** [comment]
I think @black-puppydog just moved this from 'usable' to 'barebones'. FIGHT!

**Comment by cbleslie** [comment]
Don't want to fight. Want to focus. :)

Core argument: SSB's core selling point isn't the Gathering protocol. (I mean it's good, I like it, I think it's needed). If a client's primary function should be to effectively render the basic views of the network first, then focus on ancilary features.

I am user that doesn't make use of Gatherings, but I know everyone uses basic thread conversations, and private messaging; and they need the most help for initial transition.

An aside:
Random thought; It might be interesting to move gatherings to a "calendar feature". Similar to Microsoft Teams; or Google Calendar. That or Have SSB output a feed of Gatherings in webm ical format for consumption.
/shrug

**Comment by black-puppydog** [comment]
@cbleslie the context for that move is that we're currently trying to get contracting workforce to push oasis over the edge of "we now recommend oasis to newcomers instead of patchwork."

For that, @Powersource made #554 (duplicate of this) in the barebones milestone. The reasoning (paraphrasing the way I choose to have heard it) is that all the other kids (patchX) do gatherings, and it would be bad to recommend oasis to a newcomer, and then have them miss out on something they would have gotten with a different choice.

It certainly depends on your usage, and whether or not there's a global pandemic with shelter-in-place orders going on...

**Comment by cbleslie** [comment]
@black-puppydog Kay.

**Comment by Powersource** [comment]
Yeah I'd say oasis already has the core core stuff, but I added it to the barebones milestone since for the gigs we're calling that milestone "1.0"  (I've renamed the milestone now to clarify) a.k.a rough patchwork parity and saying that the workers should primarily focus on the stuff in that milestone.

**Comment by christianbundy** [comment]
> 1.0

FWIW I think this naming is confusing, since Oasis already has a version which is > 1.0. If possible I'd love to keep using words with meaning instead of old version numbers.

**Comment by Powersource** [comment]
Ah oops didn't notice that, will try to rename to something else.
